en/lessons/basics/documentation.md

```diff
@@ -181,8 +181,8 @@ Assuming all is well and we're seeing the output above, we are now ready to set
 
 ```elixir
   def deps do
-    [{:earmark, "~> 0.1", only: :dev},
-    {:ex_doc, "~> 0.11", only: :dev}]
+    [{:earmark, "~> 1.2.5", only: :dev},
```

Comment:
Should we match these on PATCH or MINOR (`~> 1.2.5` vs `~> 1.2`)?

Reply:
Minor makes more sense for a guide, so we won't have to update that often.